lab/src/Microsoft.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace Microsoft.ServiceModel.Syndication.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using Xunit;
+    using Microsoft.ServiceModel.Syndication;
+    using System.Xml;
+    using System.IO;
+
+    public static class BasicScenarioTests
+    {
+                
+        [Fact]
+        public static void SyndicationFeed_CreateNewFeed()
+        {
+            // *** SETUP *** \\
+            SyndicationFeed sf = new SyndicationFeed("First feed on .net core ever!!", "This is the first feed on .net core ever!", new Uri("https://github.com/dotnet/wcf"));
+
+            XmlWriter xmlw = XmlWriter.Create("FirstFeedEver.xml");
+            Rss20FeedFormatter rssf = new Rss20FeedFormatter(sf);
+
+            // *** EXECUTE *** \\
+            rssf.WriteTo(xmlw);
+            xmlw.Close();
+
+            // *** VALIDATE *** \\
+            Assert.True(File.Exists("FirstFeedEver.xml"));
+
+            // *** CLEANUP *** \\
+            File.Delete("FirstFeedEver.xml");
+        }
+
+        [Fact]
+        public static void SyndicationFeed_Load_Write_Feed()
+        {
+            // *** SETUP *** \\\
+            XmlReader xmlr = XmlReader.Create(@"TestFeeds\LoadAndWrite.xml");
+            SyndicationFeed sf = SyndicationFeed.Load(xmlr);
+
+
+            // *** EXECUTE *** \\
+            //Write the same feed that was read.
+            XmlWriter xmlw = XmlWriter.Create("SyndicationFeed-Load-Write.xml");
+            Atom10FeedFormatter rs = new Atom10FeedFormatter(sf);
+            rs.WriteTo(xmlw);
+            xmlw.Close();
+
+            // *** VALIDATE *** \\
+            Assert.True(sf != null);
+            Assert.True(File.Exists("SyndicationFeed-Load-Write.xml"));
+
+            // *** CLEANUP *** \\
+            File.Delete("SyndicationFeed-Load-Write.xml");
+        }
+
+
+        [Fact]
+        public static void SyndicationFeed_Write_RSS_Atom()
+        {
+            //Test example to syndicate the release of SyndicationFeed to .net core
+
+            // *** SETUP *** \\
+
+                SyndicationFeed feed = new SyndicationFeed("Microsoft News", "<div>Most recent news from Microsoft</div>", new Uri("http://www.microsoft.com/news"), "123FeedID", DateTime.Now);
```

Comment:
Please replace all "Microsoft" with "Contoso" in this test (and any other tests if any occurrence).